src/Command/AbstractCommand.php

```diff
@@ -70,39 +75,44 @@ abstract class AbstractCommand implements CommandInterface
     use ProfilerAwareTrait;
 
     protected string|null $isolationLevel = null;
+    /** @psalm-var ParamInterface[] */
+    protected array $params = [];
     protected string|null $refreshTableName = null;
     protected Closure|null $retryHandler = null;
     private string $sql = '';
-    /** * @var ParamInterface[] */
-    protected array $params = [];
 
     public function addCheck(string $name, string $table, string $expression): static
     {
         $sql = $this->queryBuilder()->addCheck($name, $table, $expression);
+
         return $this->setSql($sql)->requireTableSchemaRefresh($table);
     }
 
     public function addColumn(string $table, string $column, string $type): static
     {
         $sql = $this->queryBuilder()->addColumn($table, $column, $type);
+
         return $this->setSql($sql)->requireTableSchemaRefresh($table);
     }
 
     public function addCommentOnColumn(string $table, string $column, string $comment): static
     {
         $sql = $this->queryBuilder()->addCommentOnColumn($table, $column, $comment);
+
```

Comment:
It looks terrible. We make 3 out of 2 understandable lines of code. As a result, the hand gets tired of scrolling through the code.
This is contrary to common sense.